css/acquia_lift.personalize.theme.css

```diff
@@ -1,9 +1,9 @@
-/* Page variation edit toggle in unified navigation bar */
```

Comment:
Okay. Removed until the next section "QTips". This also means the image _edit-toggle-disabled.png_ is no longer referenced. Is this expected? If so, should I remove this image?